### ISSUE TITLE
pex uninstall?

### ISSUE THREAD

**Issue by jmagnusson** [petere/pex#5]
Is there any way to uninstall an extension, say if I want to downgrade to an older version?

**Comment by petere** [comment]
Downgrading should work without uninstalling, but just plain uninstall would be a good feature addition.

**Comment by jmagnusson** [comment]
Agreed, I was surprised to not find one. I think it's almost taken for granted in package managers.
